numba_cuda/numba/cuda/tests/nrt/test_nrt.py

```diff
@@ -76,6 +117,31 @@ def g(out_ary):
 
         self.assertEqual(out_ary[0], 1)
 
+    def test_nrt_detect_linked_ptx_file(self):
+        src = """;
+                 extern "C" __device__ void* NRT_Allocate(size_t size);
```

Comment:
Would this be an opportunity to check if `get_include` work as intended?

```suggestion
                 #include <{get_include()}>
```

Reply:
The general idea is that user can depend on Numba's C++ declaration of the NRT APIs, but not to depend on their own, right?